### PR TITLE
Oblig2 document

### PR DESCRIPTION
**Pull request by KimAndreG**

### CODE REVIEW

File: Deliverables/ObligatoriskOppgave2.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+
+
+BUGS:
+
+- Vi er ikke ferdig å implementere multiplayer så mye kræsjer hvis man prøver å enten hoste eller joine et spill :
+    - Timer funker ikke helt som den skal slik at “gameState” ikke blir oppdatert fordi ikke ferdig implementert enda
+    
+
+#Oppgave 3
```
Comment (mrtineide) [inline Deliverables/ObligatoriskOppgave2.md]: Need space between # and "oppgave 3" text

**Review by mrtineide — CHANGES_REQUESTED** [review]
Need more things added and fixed formatting

File: Deliverables/ObligatoriskOppgave2.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+**Brukerhistorier** - Bli bedre på å planlegge og bruke brukerhistorier før vi starter arbeidet.
+
+**Jevnere Arbeid** -  Sette av mer til i uken til å jobbe med prosjektet.
+
+**Fordele commits** - Fordele commits mer jevnt blant gruppen slik at arbeidsfordelingen ser mer realistisk ut.
+
+
+
+#Oppgave 2)
```
Comment (mrtineide) [inline Deliverables/ObligatoriskOppgave2.md]: Need space between # and "oppgave 3" text, and need to make it a H2 by adding

**Review by mrtineide — CHANGES_REQUESTED** [review]

**Review by mrtineide — APPROVED** [review]
:shipit:

**Review by TheLimeFreak — APPROVED** [review]
Looks good now

**Review by OdinBjerga — APPROVED** [review]
Approved!

**Pull request merged**